beetsplug/musicbrainz.py

```diff
@@ -370,6 +371,10 @@ def _merge_pseudo_and_actual_album(
 
 
 class MusicBrainzPlugin(MetadataSourcePlugin):
+    @cached_property
+    def genres_field(self) -> str:
+        return f"{config['musicbrainz']['genres_tag'].get()}-list"
```

Comment:
To remove any potential confusion both for users and for us, I think it would be a good idea to validate the value here:

```suggestion
        return self.config["genres_tag"].as_choice(["genre", "tag"])
```